doc/nrf/device_guides/working_with_fem.rst

```diff
@@ -390,6 +378,41 @@ To use nRF21540 in SPI or mixed mode, complete the following steps:
    Replace the SPI bus according to your hardware design.
 
 #. Create alternative pinctrl entries for SPI3 and replace the ``pinctrl-N`` and ``pinctrl-names`` properties.
+#. On nRF53 devices, the devicetree nodes described above must be added to the network core.
+   For the application core, a GPIO forwarder node must be added to the devicetree file:
+
+   .. code-block:: devicetree
+
+      &gpio_fwd {
+         nrf21540-gpio-if {
+            gpios = <&gpio0 11 0>,   /* tx-en-gpios */
+                    <&gpio0 12 0>,   /* rx-en-gpios */
+                    <&gpio0 13 0>,   /* pdn-gpios */
+                    <&gpio0 14 0>,   /* ant-sel-gpios */
+                    <&gpio0 15 0>;   /* mode-gpios */
+         };
+         nrf21540-spi-if {
+            gpios = <&gpio1 16 0>,   /* cs-gpios */
+                    <&gpio1 15 0>,   /* SPIM_SCK */
+                    <&gpio1 14 0>,   /* SPIM_MISO */
+                    <&gpio1 13 0>;   /* SPIM_MOSI */
+         };
+      };
+
+   The pins defined in the GPIO forwarder node in the application core's devicetree file must match the pins defined in the FEM nodes in the network core's devicetree file.
+
+#. On nRF53 devices, SPIM0 and UARTE0 are mutually exclusive AHB bus masters on the network core as follows from `Product Specification <nRF5340 Product Specification_>`_, Section 6.4.3.1, Table 22.
+   As a result, they cannot be used simultaneously.
+   For the SPI part of the nRF21540 interface to be functional, the UARTE0 node must be disabled in the network core's devicetree file.
+
+   .. code-block:: devicetree
+
+      &uart0 {
+         status = "disabled";
+      };
```

Comment:
This is only relevant if `SPIM0` is used for FEM SPI. The devicetree snippet further up uses `SPIM3`, so maybe it is enough to point this out in the nRF21540 EK section?

Reply:
It is always relevant for nRF53 devices where there's only one instance of `SPIM` and `UART` peripherals on the network core. The example you mentioned uses `SPIM3` available on nRF52 series. Do you think that further clarification about SPIM and UART being mutually exclusive is needed?

Reply:
I think this is fine as it is.